source/dub/packagesupplier.d

```diff
@@ -322,7 +324,18 @@ package alias FallbackPackageSupplier = AutoImplement!(AbstractFallbackPackageSu
 private template fallback(T, alias func)
 {
 	enum fallback = q{
-		scope (failure) return m_fallback.%1$s(args);
+		import std.range : back, dropBackOne;
+		scope (failure)
+		{
+			foreach (m_fallback; m_fallbacks.dropBackOne)
+			{
+				try
+					return m_fallback.%1$s(args);
+				catch(Throwable)
+				    assert(1);
```

Comment:
Isn't `{}` what you should use for an empty statement?

Reply:
More importantly, `Throwable` is a bad idea, must be `Exception` - logging the error with `logDiagnostic` or maybe just `logDebug` could also be important for remote debugging .